pypeit/scripts/ql_keck_mosfire.py

```diff
@@ -213,6 +214,26 @@ def run_pair(A_files, B_files, caliBrate, spectrograph, det, parset, show=False,
                                       slits=copy.deepcopy(caliBrate.slits))
     return spec2DObj_A, spec2DObj_B
 
+def find_single_file(file_pattern):
```

Comment:
Nice, should we move this to core.utils. It looks like other routines would benefit.